specviz/data/ui/equation_editor.ui

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like a duplicate? There is another one: `specviz/plugins/arithmetic/equation_editor.ui`

Reply:
Another duplicate, it's gone now.